validator/client/runner.go

```diff
@@ -15,6 +16,10 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+// time to wait before trying to reconnect with the eth1 node.
+var backOffPeriod = 15 * time.Second
+var connectionIssueMsg = ", beacon chain may be offline..."
```

Comment:
This message string is a bit awkward starting with the ",". Also if it is an error, we should name it errConnectionIssue

Reply:
+1 there is a common pattern to naming errors:
```
errConnectionIssue
```
etc